### ISSUE TITLE
Error trying to fetch attestations

### ISSUE THREAD

**Issue by bodgit** [terraform-linters/tflint#2460]
### Summary

The latest release seems to have introduced an error when trying to access the attestations API.

I’m running tflint in a private GH enterprise account and we also have an internal tflint plugin hosted in an internal private repo, which has always been verified via PGP, there are no attestations.

Now my tflint GH actions fail with the following:
```
Installing "aws" plugin...
Installed "aws" (source: github.com/terraform-linters/tflint-ruleset-aws, version: 0.45.0)
Installing "<myplugin>" plugin...
Failed to install a plugin; Failed to download artifact attestations: GET https://api.github.com/repos/<org>/<myplugin>/attestations/sha256:…: 403 Resource not accessible by integration []
Error: Process completed with exit code 1.
```
My workflow doesn’t have access to the attestation API and actually I can’t grant it access because of actions/create-github-app-token#310.

Ideally tflint should handle/ignore a 403 error and assume no attestations available.

Downgrading back to 0.60.0 gets things working again.

### Command

tflint —init

### Terraform Configuration

```terraform
n/a
```

### TFLint Configuration

```HCL
plugin "terraform" {
  enabled = true
  preset  = "all"
}

plugin "aws" {
  enabled = true
  version = "0.45.0"
  source  = "github.com/terraform-linters/tflint-ruleset-aws"
}

plugin "myplugin" {
  enabled = true

  version = "1.0.3"
  source  = "github.com/<org>/<myplugin>"

  signing_key = <<-KEY
  -----BEGIN PGP PUBLIC KEY BLOCK-----
  …
  -----END PGP PUBLIC KEY BLOCK-----
  KEY
}
```

### Output

```Shell
Installing "aws" plugin...
Installed "aws" (source: github.com/terraform-linters/tflint-ruleset-aws, version: 0.45.0)
Installing "<myplugin>" plugin...
Failed to install a plugin; Failed to download artifact attestations: GET https://api.github.com/repos/<org>/<myplugin>/attestations/sha256:…: 403 Resource not accessible by integration []
Error: Process completed with exit code 1.
```

### TFLint Version

0.61.0

### Operating System

- [x] Linux
- [ ] macOS
- [ ] Windows
